.prod/uwsgi.ini

```diff
@@ -0,0 +1,14 @@
+[uwsgi]
+http-socket = :8081
+chdir = /app
+module = apartment_application_service.wsgi
+static-map = /static=/app/static
+uid = nobody
+gid = nogroup
+buffer-size = 32768
+master = 1
+processes = 2
+threads = 2
+; don't log readiness and healthz endpoints
+route = ^/readiness$ donotlog:
+route = ^/healthz$ donotlog:
```

Comment:
We don't have these endpoints (yet?), so it might be okay to leave these out. 🙂

Reply:
This is removed as well, we don't have these at least yet.